ml-prims/test/metrics.cu

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#include "metrics/metrics.h"
+#include "score/metrics.h"
```

Comment:
If we are going to change the directory here, we may as well change the namespace and file, since R^2 is not a true metric. I would change this to `score/scores.h` and we can put actual metrics in `scores/metrics.h`